src/ui.h

```diff
@@ -0,0 +1,31 @@
+#ifndef UI_H
+#define UI_H
+
+#include "block.h"
+#include "board.h"
+#include <vector>
+
+enum ColorSet
+{
+    DEFAULT_COLOR,
+    MAIN_BORDER_COLOR,
+    TETRIS_BORDER_COLOR
+};
+
+class UI
+{
+    uint columns;
+    uint rows;
+    
+    std::vector<std::vector<std::vector<bool>>> blockMatrices;
```

Comment:
Weź pod uwagę, że jeszcze powinieneś pamiętać kolory klocków - raczej zrób ` std::vector<Board>`